ocaml/sdk-gen/common/CommonFunctions.ml

```diff
@@ -111,137 +93,188 @@ and is_method_static message =
   | {param_type= ty; _} :: _ ->
       not (ty = Ref message.msg_obj_name)
 
-and get_method_params_list message =
-  if is_method_static message then
-    message.msg_params
-  else
-    List.tl message.msg_params
+and is_setter message =
+  String.length message.msg_name >= 3 && String.sub message.msg_name 0 3 = "set"
```

Comment:
I'm not a big fan of using `String.sub` for sub-string matching. How about `Astring.String.is_prefix`